tee-worker/cli/lit_set_heartbeat_timeout.sh

```diff
@@ -35,7 +35,7 @@ NODEURL=${NODEURL:-"ws://127.0.0.1"}
 WORKER1PORT=${WORKER1PORT:-2000}
 WORKER1URL=${WORKER1URL:-"wss://127.0.0.1"}
 
-CLIENT_BIN=${CLIENT_BIN:-"./bin/litentry-cli"}
+CLIENT_BIN=${CLIENT_BIN:-"../bin/litentry-cli"}
```

Comment:
Why did it work before?

Reply:
Here is the default value. In test, there is input parameter to overwrite it.